extensions/timeout_heap_test.go

```diff
@@ -40,7 +41,7 @@ var _ = Describe("[Unit]", func() {
 		It("should push a new timeout node to heap", func() {
 			th := NewTimeoutHeap(config)
 
-			var deviceToken string = "string_device"
+			var deviceToken = "string_device"
```

Comment:
consider using `deviceToken := "string_device"`, you generally don't use the `var` keyword in go unless necessary.

Reply:
Good deal, will remove the var keyword